html-css/week1/index.html

```diff
@@ -215,16 +229,18 @@ <h3>Git</h3>
           </div>
         </div>
       </div>
+      <a href="./#projects">
       <img
         src="./assets/arrow.png"
         alt="Arrow icon"
         class="icon arrow"
-        onclick="location.href='./#projects'"
-      />
+        />
+      </a>
+    
     </section>
     <section id="projects">
       <p class="section__text__p1">Browse My Recent</p>
-      <h1 class="title">Projects</h1>
+      <h2 class="title">Projects</h2>
       <div class="experience-details-container">
         <div class="about-containers">
           <div class="details-container color-container">
```

Comment:
There are many container divs here. Not sure if you couldn't get away with one or two less, but my bigger concern is that the Projects cause horizontal scrolling. Is this intentional, or would you prefer to style them 2 per row for example?

Reply:
Is not intentional, I did not observed until now as I have opened the project on a big screen and I couldn't see the  horizontal scrolling. I don't prefer to be 2 in a row, and in this case I think I need to change the CSS  style, as in body the max-width to 100% and overflow-x : hidden; In rest  I don't know if I can remove any div from here as all I use for styling. :)